ipatests/prci_definitions/nightly_latest_sssd.yaml

```diff
@@ -0,0 +1,249 @@
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{sssd-fedora/build_url}'
+        update_packages: True
+        copr: '@sssd/nightly'
+        test_suite: test_integration/test_idviews.py
+        template: *ci-master-latest
+        timeout: 7200
+        topology: *ad_master_2client
+
+  sssd-fedora/test_netgroup:
+    requires: [sssd-fedora/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{sssd-fedora/build_url}'
+        update_packages: True
+        copr: '@sssd/nightly'
+        test_suite: test_integration/test_netgroup.py
+        template: *ci-master-latest
+        timeout: 3600
+        topology: *master_1repl
+
+  sssd-fedora/test_nfs_TestIpaClientAutomountFileRestore:
+    requires: [sssd-fedora/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{sssd-fedora/build_url}'
+        update_packages: True
+        copr: '@sssd/nightly'
+        test_suite: test_integration/test_nfs.py::TestIpaClientAutomountFileRestore
+        template: *ci-master-latest
+        timeout: 3600
+        topology: *master_3client
+
+  sssd-fedora/test_otp:
+    requires: [sssd-fedora/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{sssd-fedora/build_url}'
+        update_packages: True
+        copr: '@sssd/nightly'
+        test_suite: test_integration/test_otp.py
+        template: *ci-master-latest
+        timeout: 3600
+        topology: *master_1repl
+
+  sssd-fedora/test_replica_promotion_TestReplicaPromotionLevel1:
+    requires: [sssd-fedora/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{sssd-fedora/build_url}'
+        update_packages: True
+        copr: '@sssd/nightly'
+        test_suite: test_integration/test_replica_promotion.py::TestReplicaPromotionLevel1
+        template: *ci-master-latest
+        timeout: 7200
+        topology: *master_1repl
+
+  sssd-fedora/test_replica_promotion_TestUnprivilegedUserPermissions:
+    requires: [sssd-fedora/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{sssd-fedora/build_url}'
+        update_packages: True
+        copr: '@sssd/nightly'
+        test_suite: test_integration/test_replica_promotion.py::TestUnprivilegedUserPermissions
+        template: *ci-master-latest
+        timeout: 7200
+        topology: *master_1repl
+
+  sssd-fedora/test_replica_promotion_TestWrongClientDomain:
+    requires: [sssd-fedora/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{sssd-fedora/build_url}'
+        update_packages: True
+        copr: '@sssd/nightly'
+        test_suite: test_integration/test_replica_promotion.py::TestWrongClientDomain
+        template: *ci-master-latest
+        timeout: 7200
+        topology: *master_1repl
+
+  sssd-fedora/test_smb:
+    requires: [fedora-fedora/build]
```

Comment:
This should be:
    requires: [sssd-fedora/build]
and the definition also needs:
         update_packages: True
         copr: '@sssd/nightly'

Reply:
Fixed. Thanks for the feedback.